xml/System.Threading/Interlocked.xml

```diff
@@ -1559,8 +1559,14 @@ This method wraps a call to [FlushProcessWriteBuffers](/windows/desktop/api/proc
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
+        <summary>Defines a memory fence that blocks speculative execution past this point until pending reads and writes are complete.</summary>
+        <remarks>  
+          <format type="text/markdown"><![CDATA[  
+
+The `SpeculationBarrier` method defines a memory fence (an `mfence` instruction on x86 and amd64 processors) that blocks *speculative* execution past this point until pending reads and writes are complete. 
+
+       ]]></format>
```

Comment:
I have a slight impression that too many spaces here cause an issue for loc but I could be wrong 

```suggestion
 ]]></format>
```